mysql-test/t/percona_bug_ps5541.test

```diff
@@ -0,0 +1,47 @@
+-- source include/big_test.inc
```

Comment:
If I understand correctly, the test does *not* require redo log encryption itself to be enabled, correct? In which case this scenario should be test by existing innodb.percona_changed* testcases, please check, whether the new testcase is required at all

Reply:
That's right, encryption is not needed - but a huge amount of data is. If I insert fewer records than what we have in this test case, the issue won't show up without this commit. And with this amount, it runs for 15-20 minutes on a debug build.

Reply:
OK, then the existing bitmap testcases have predetermined name prefix (percona_changed_page_bmp in general, percona_changes_pages for I_S queries like this one), so please rename to percona_changed_pages_big.